auth/token_test.go

```diff
@@ -81,8 +85,8 @@ func TestOIDCClaims_getClaimAsStringSlice(t *testing.T) {
 			name:       "single string",
 			claimValue: "myValue",
 			claimArg:   "testClaim",
-			want:       nil,
-			wantErr:    true,
+			want:       []string{"myValue"},
```

Comment:
Could you please add another test case where claim value is of type `[]interface`?